tests/core/structured_vespa_index/test_structured_vespa_schema.py

```diff
@@ -149,3 +149,66 @@ def _remove_whitespace_in_schema(self, schema: str) -> str:
         schema = re.sub(r'^\s+', '', schema, flags=re.MULTILINE)
 
         return schema
+
+
+class TestStructuredVespaSchemaMethods(MarqoTestCase):
```

Comment:
I would argue that you don't need these tests.  You can add full schema tests like the ones you fixed above to cover these cases if you find these necessary. As a rule of thumb, we don't write unit tests for private methods -- we only test the public interface. If you find it impossible to get good coverage without unit-testing private methods, then chances are your design isn't cohesive (which is not the case, your design is fine here). Check this out also https://chat.openai.com/share/d6c43184-1ad5-49b5-b209-e659bdb68c33

Reply:
Got this, I'll note this for future private methods. That being said, should I just keep them in since I already wrote them? Or would you prefer them removed?

Reply:
I would argue though that it's cumbersome to make 5 full new schemas just to test the output of 5 different cases of `_generate_max_similarity_expression`. It seems like a lot of wasted space.